src/components/atoms/Box/Box.component.tsx

```diff
@@ -5,9 +5,9 @@ import { CardInterface } from 'antd/lib/card'
 import { PreviewProps } from '@/types/preview'
 import { ComponentProps } from '@/types/component'
 
-export interface CardProps extends PreviewProps, ComponentProps<AntCardProps> {}
+export interface BoxProps extends PreviewProps, ComponentProps<AntCardProps> {}
```

Comment:
why we rename `Card` to `Box`?

Reply:
As discussed on this thread

https://iruuza-inc.slack.com/archives/C02MH3VMBTJ/p1649224989079219